plugin-ci/build/action.yaml

```
@@ -23,9 +23,20 @@ runs:
         fi
 
     - uses: actions/upload-artifact@83fd05a356d7e2593de66fc9913b3002723633cb # v3.1.1
+      if: ${{ github.event_name != 'schedule' }}
       with:
         name: dist
         path: |
           dist/*.tar.gz
           dist/release-notes.md
         if-no-files-found: error
+
+    - uses: actions/upload-artifact@83fd05a356d7e2593de66fc9913b3002723633cb # v3.1.1
+      if: ${{ github.event_name == 'schedule' }}
+      with:
+        name: dist
+        path: |
+          dist/*.tar.gz
+          dist/release-notes.md
+        if-no-files-found: error
+        retention-days: 5
```

I'm wondering if instead of introducing the duplicated block, we can do something with Action expressions like:

```suggestion
        retention-days: ${{ github.event_name == 'schedule' ? 5 : 90 }}
```

---

https://github.com/actions/runner/issues/409#issuecomment-727565588

```suggestion
        retention-days: ${{ github.event_name == 'schedule' && 5 || 90 }}
```